.gitignore

```diff
@@ -1,6 +1,8 @@
 # dependencies
 /node_modules
+
 /dist
+cpToken.json
```

Comment:
Shall we remove this, hence it is not going to be used in the future

Reply:
Even though this will remove later, current code is referring this. Once we get rid of this flow with proper authentication, we can remove this. Let's keep this until then.